active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,622 @@
+
+- For scenarios where a persistent notification is not needed, e.g. displaying information in context of 
+a specific UI element, a 
+[Flyout](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/flyouts) 
+is a better option. 
+- For scenarios where the application is confirming a user action, showing information the user ***must*** read, use a
+[ContentDialog](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/dialogs).
+  - Additionally, if a status change to the app is so severe that it needs to block all further ability for the user to interact with the app, use a ContentDialog.
+- For scenarios where a notification is a transient teaching moment, a 
+[TeachingTip](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/teaching-tip) 
+is a better option.
+
+
+For more info about choosing the right notification control, see the 
+[Dialogs and Flyouts](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/)
+article.
+
+
+# Examples
+
+## Create an InfoBar
+The XAML below describes an inline InfoBar with the default styling for an informational notification. 
+An info bar can be created anywhere in the element tree or code behind and will expand 
+to fill the width of the layout control it resides in.
+
+
+By default, the InfoBar will not be visible. Set the IsOpen property to true in the XAML or code behind
+to display the InfoBar.
+
+
+XAML
+```xml
+<controls:InfoBar x:Name="UpdateAvailableNotification"
+    Title="Update available."
+    Message="Restart the application to apply the latest update.">
+</controls:InfoBar>
+```
+C#
+```C#
+public MainPage()
+{
+    this.InitializeComponent();
+
+    if(IsUpdateAvailable())
+    {
+        UpdateAvailableNotification.IsOpen = true;
+    }
+}
+```
+
+![A mockup of an InfoBar in the default state with a close button and a message](images/Default_TitleMessage.png)
+
+## Using pre-defined severity levels
+The type of the info bar can be set via the Severity property to automatically set a consistent status color, icon, and assistive technology settings dependent on the criticality of the notification. If no Severity is set, the default informational styling is applied.
+
+
+XAML
+```xml
+<controls:InfoBar x:Name="SubscriptionExpiringNotification"
+    Severity="Warning"
+    Title="Your subscription is expiring in 3 days"
+    Message="Renew your subscription to keep all functionality" />
+```
+![A mockup of an InfoBar in a Warning state with a close button and a message](images/Warning_TitleMessage.png)
+
+## Programmatic dismiss in InfoBar
+An InfoBar can be dismissed by the user via the close button or programmatically. 
+If the notification is required to be in view until the status is resolved and you would like to remove 
+the ability for the user to dismiss the info bar, you can set the IsUserDismissable property to false.
```

Comment:
"Close" is a more standard term for this action than "dismiss". And this should follow the "Can" pattern for consistency. So "CanClose" rather than "IsUserDismissable". (For example similar to [TextBox.CanUndo](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.TextBox.CanUndo).)